bin/compile

```diff
@@ -18,32 +31,57 @@ function indent() {
   esac
 }
 
-BUILD_DIR=${1:-}
-CACHE_DIR=${2:-}
-ENV_DIR=${3:-}
+topic "Installing Chrome for Testing"
 
-BIN_DIR=$BUILD_DIR/.chromedriver/bin
-mkdir -p $BIN_DIR
+INSTALL_DIR="$BUILD_DIR/.chrome-for-testing"
+BIN_DIR="$INSTALL_DIR/bin"
+mkdir -p "$BIN_DIR"
 
-if [ -f $ENV_DIR/CHROMEDRIVER_VERSION ]; then
-  VERSION=$(cat $ENV_DIR/CHROMEDRIVER_VERSION)
-  topic "Using chromedriver v${VERSION} (set by the env var 'CHROMEDRIVER_VERSION')"
+# Detect if the old config var is still used
+if [ -f "$ENV_DIR/CHROMEDRIVER_VERSION" ]; then
+  error "This buildpack no longer supports CHROMEDRIVER_VERSION config var and must be removed to continue, because this buildpack now installs compatible versions of Chrome & Chromedriver. GOOGLE_CHROME_CHANNEL can be used to set Stable (default), Beta, Dev, or Canary."
+fi
+
+# Detect if Chrome is already installed
+if which chrome; then
+  error "Chrome is already installed. This buildpack now install a compatible version of Chrome. Please, remove the other Chrome installation."
```

Comment:
```suggestion
  error "Chrome is already installed. This buildpack now installs a compatible version of Chrome. Please, remove the other Chrome installation. Chrome may have been installed by another buildpack (such as heroku/heroku-buildpack-google-chrome buildpack), which should no longer be used with this buildpack."
```

Users may not know what installed Chrome right away, but we already do (heroku-buildpack-google-chrome or it's forks). We could be more direct about instructing users how to fix the issue.